src/styles/index.scss

```diff
@@ -1,3 +1,119 @@
+$white: #fff;
+$lightBlue: #00acdc;
+$gray: #616070;
+$lightGray: #f3f3f3;
+$darkBlue: #060b35;
+
 body {
   margin: 0;
 }
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid $lightGray;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background: $white;
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+
+  &__image {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    font-size: 12px;
+    margin: 0 0 4px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $darkBlue;
+  }
+
+  &__code {
+    font-size: 10px;
+    color: $gray;
+    margin-bottom: 16px;
+  }
+
+  &__review-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 24px;
+  }
+
+  &__price-container {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+
+  &__price-word {
+    font-size: 12px;
+    line-height: 18px;
+    color: $gray;
+  }
+
+  &__price-number {
+    font-size: 16px;
+    line-height: 18px;
+    font-weight: bold;
+    color: $darkBlue;
+  }
+
+  &__link {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    background-color: $lightBlue;
+    box-sizing: border-box;
+    height: 40px;
+    border: 1px solid $lightBlue;
+    border-radius: 5px;
+
+    font-size: 14px;
+    line-height: 16px;
+    color: #fff;
+    font-weight: 700;
+    text-decoration: none;
+    text-transform: uppercase;
+
+    &:hover {
+      background-color: $white;
+      color: $lightBlue;
+    }
+  }
+}
+
+.stars {
```

Comment:
if you're familiar with file importing I'd recommend you to create root file (index.scss) 

For stars you can create stars.scss file
For card - card.scss
For variables - vars.scss
For common styles (like `body`/`*`/`html`) - common.scss

After these actions you can just import files
vars.scss, common.scss, stars.scss, card.scss to index.scss and connect index.scss to index.html